libraries/AP_UAVCAN/AP_UAVCAN.cpp

```diff
@@ -79,6 +81,15 @@ const AP_Param::GroupInfo AP_UAVCAN::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("SRV_RT", 4, AP_UAVCAN, _servo_rate_hz, 50),
     
+    // @Param: VSER_P
```

Comment:
I think we need to discuss how we will have parameters for multiple remote uarts. I'd love to see a UART adapter board built with several UARTs (maybe 4?) all available over CAN.

Reply:
I would suggest we create a new packet for this or, since uavcan.tunnel.broadcast is very new, we could add it to that. Replace "protocol" with "channel" or maybe have both in there. @pavel-kirienko what do you think about that?

Reply:
Eh, let's add a channel ID. This is getting out of hand, this whole tunneling thing, but I have no better ideas atm.